examples/visualization/brain.py

```diff
@@ -0,0 +1,65 @@
+"""
+.. _ex-brain:
+
+===============================
+Plotting with ``mne.viz.Brain``
+===============================
+
+In this example, we'll show how to use :class:`mne.viz.Brain`.
+"""
+# Author: Alex Rockhill <aprockhill@mailbox.org>
+#
+# License: BSD-3-Clause
+
+# %%
+
+import os.path as op
+import matplotlib.pyplot as plt
+
+import mne
+from mne.datasets import sample
+
+print(__doc__)
+
+data_path = sample.data_path()
+sample_dir = op.join(data_path, 'MEG', 'sample')
+subjects_dir = op.join(data_path, 'subjects')
+evoked = mne.read_evokeds(op.join(sample_dir, 'sample_audvis-ave.fif'))[0]
+trans = mne.read_trans(op.join(sample_dir, 'sample_audvis_raw-trans.fif'))
+stc = mne.read_source_estimate(op.join(sample_dir, 'sample_audvis-meg'))
+
+brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
+                      alpha=0.5, cortex='low_contrast')
```

Comment:
```suggestion
# %%
# Plot a brain image
# ------------------
#
# First we plot a brain. This requires a directory with MRI? information.
# MNE provides functionality to automatically retrieve standard head images.
# In this example we use the ``sample`` data which is?
# Another common head used in the MNE documentation is fsaverage, which is?

data_path = sample.data_path()
subjects_dir = op.join(data_path, 'subjects')

brain = mne.viz.Brain('sample', subjects_dir=subjects_dir,
                      alpha=0.5, cortex='low_contrast')
```

Reply:
I pushed the evoked and stc imports below as they may just confuse people who wish to just plot a brain. I think keep it simple, and then build up complexity.